src/components/shared/StackSettings.js

```diff
@@ -145,6 +145,7 @@ class StackSettings extends PureComponent<Props> {
               <label>
                 Summarize:{' '}
                 <select
+                  className="stackSettingsSelect"
```

Comment:
Snapshot of this change is slipped through the previous commit. It would be good to keep the code and snapshot change in the same commit.